app-text/mupdf/mupdf-1.20.0.ebuild

```diff
@@ -69,6 +69,10 @@ src_prepare() {
 		-e "1iverbose = yes" \
 		-e "1ibuild = debug" \
 		-i Makerules || die "Failed adding build variables to Makerules in src_prepare()"
+
+	# Substitute MuPDF version in pkgconfig file
+	sed -e "s/Version: \(.*\)/Version: ${PV}/" \
```

Comment:
`platform/debian/mupdf.pc` comes from `"${FILESDIR}"/${PN}-1.10a-add-desktop-pc-xpm-files.patch`. I think the sed comment should state the fact. As otherwise, I had asked you if you reported the issue upstream. But upstream does not ship a pkgconfig file.

Reply:
Thanks. That makes sense; I adjusted it.